src/ServicePulse.Host/vue/src/components/failedmessages/PendingRetries.vue

```diff
@@ -247,8 +247,8 @@ onUnmounted(() => {
 
 onMounted(() => {
   let cookiePeriod = cookies.get("pending_retries_period");
-  if (typeof cookiePeriod === "undefined") {
-    cookiePeriod = periodOptions[0];
+  if (typeof cookiePeriod === "undefined" || cookiePeriod === "" || cookiePeriod === null) {
```

Comment:
This will cover all three of those checks https://developer.mozilla.org/en-US/docs/Glossary/Falsy

```suggestion
  if (!cookiePeriod) {
```